package.json

```diff
@@ -31,64 +28,52 @@
   "author": "HMRC",
   "license": "Apache-2.0",
   "dependencies": {
-    "backstopjs": "hmrc/BackstopJS",
-    "browser-sync": "^2.4.0",
+    "acorn": "^7.4.0",
+    "browser-sync": "^2.26.12",
     "browserify": "^14.4.0",
     "browserify-shim": "^3.8.14",
     "bundle-collapser": "^1.2.1",
-    "datatables": "1.10.11",
-    "del": "^1.1.1",
-    "fingerprintjs": "^0.5.3",
```

Comment:
Are you sure this isn't being used? I know it was used quite a lot in the past, at least in older versions of AF

Reply:
Which dependency are you referring to?

Reply:
fingerprintjs

Reply:
I think the code is checked in, it's not brought in via node_modules : https://github.com/hmrc/assets-frontend/blob/master/assets/javascripts/modules/mdtpdf.js

Reply:
Ok fair enough. Guessing it was originally from node_modules and then pulled in and modified.